drivers/include/sht3x.h

```diff
@@ -0,0 +1,168 @@
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** possible I2C slave addresses */
+#define SHT3X_I2C_ADDR_1    (0x44)  /**< ADDR pin connected to GND/VSS */
+#define SHT3X_I2C_ADDR_2    (0x45)  /**< ADDR pin connected to VDD */
+
+/**
+ * @brief   Driver error codes (returned as negative values)
+ */
+typedef enum {
+    SHT3X_OK,                       /**< no error */
+    SHT3X_ERROR_I2C,                /**< I2C communication failure */
+    SHT3X_ERROR_CRC,                /**< CRC check failed*/
+    SHT3X_ERROR_STATUS,             /**< sensor has wrong status */
+    SHT3X_ERROR_MEASURE_CMD_INV,    /**< measurement command not executed */
+} sht3x_error_codes;
+
+/**
+ * @brief   SHT3x measurement modes
+ */
+typedef enum {
+    SHT3X_SINGLE_SHOT = 0,  /**< single shot measurement */
+    SHT3X_PERIODIC_0_5_MPS, /**< periodic with 0.5 measurements per second (mps) */
+    SHT3X_PERIODIC_1_MPS,   /**< periodic with   1 measurements per second (mps) */
+    SHT3X_PERIODIC_2_MPS,   /**< periodic with   2 measurements per second (mps) */
+    SHT3X_PERIODIC_4_MPS,   /**< periodic with   4 measurements per second (mps) */
+    SHT3X_PERIODIC_10_MPS   /**< periodic with  10 measurements per second (mps) */
+} sht3x_mode_t;
+
+
+/**
+ * @brief   SHT3x repeatability levels
+ *
+ * The stated repeatability is 3 times the standard deviation of multiple
+ * consecutive measurements at the stated repeatability and at constant
+ * ambient conditions. It is a measure for the noise on the physical
+ * sensor output. Different measurement modes allow for high/medium/low
+ * repeatability. [Datasheet SHT3x-DIS]
+ *
+ * The configured repeatability level determines the time required for
+ * a measurement:
+ *
+ * Reapeatability  | T typ.  | RH typ. | Maximum Measurement Duration
+ * :---------------|:-------:|:-------:|:---------------------------:
+ * high            | 0.04    | 0.08    | 15.5 ms
+ * medium          | 0.08    | 0.15    | 6.5 ms
+ * low             | 0.15    | 0.21    | 4.5 ms
+ */
+typedef enum {
+    SHT3X_HIGH = 0, /**< high repeatability */
+    SHT3X_MEDIUM,   /**< medium repeatability */
+    SHT3X_LOW       /**< low repeatability */
+} sht3x_repeat_t;
+
+
+/**
+ * @brief   SHT3x device initialization parameters
+ */
+typedef struct {
+    i2c_t          i2c_dev;   /**< I2C device               */
+    uint8_t        i2c_addr;  /**< I2C address              */
+    sht3x_mode_t   mode;      /**< measurement mode used    */
+    sht3x_repeat_t repeat;    /**< repeatability level used */
+} sht3x_params_t;
+
+/**
+ * @brief   SHT3x sensor device data structure
+ */
+typedef struct {
+
+    i2c_t           i2c_dev;         /**< I2C device  */
+    uint8_t         i2c_addr;        /**< I2C address */
+
+    sht3x_mode_t    mode;            /**< measurement mode used    */
+    sht3x_repeat_t  repeat;          /**< repeatability level used */
+
+    bool            meas_started;    /**< indicates whether measurement is started */
```

Comment:
`#include <stdbool.h>` is missing

Reply:
Thanks for the hint. I didn't realize that on my platform.